README.md

```diff
@@ -63,7 +63,7 @@ docker run -d --net bookstack_nw  \
 -e DB_USERNAME=bookstack \
 -e DB_PASSWORD=secret \
 -p 8080:80 \
- solidnerd/bookstack:0.25.0
+ solidnerd/bookstack:0.25.1
```

Comment:
@solidnerd, non existent tag :( please push documented tag, latest is bad option for versioning.

```
Error response from daemon: manifest for solidnerd/bookstack:0.25.1 not found
```